.github/workflows/publish.yml

```diff
@@ -0,0 +1,47 @@
+name: Package
+on:
+  release:
+    types: [released, edited]
+        
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Check out the repo
+        uses: actions/checkout@v2
+      - 
+        name: Determine Short SHA
+        run: echo ${{ github.sha }} | tail -c 8 | (read; echo SHORT_SHA=$REPLY) >> $GITHUB_ENV
+      -
+        name: Sanitize Repo Name for Tagging
+        run: echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]' | (read; echo REPO_LOWER=$REPLY) >> $GITHUB_ENV
+      - 
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - 
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.CLOUDY_USERNAME }}
```

Comment:
@claymcleod This will need to be set in the repo settings

Reply:
@jordan-rash both secrets have been added and @stjudecloud-cloudy should have access to push that docker image. Give it a shot.

Reply:
@claymcleod I don't have access to merge on this repo. 
I will be be standby to make sure it all works.